.env.dist

```diff
@@ -1,70 +1,74 @@
+##### (Example: BACKUP_CODE_SALT=af21c49e35f01c1ec4d465daf75098e07a04c7ed)
 BACKUP_CODE_SALT=salt
-BACKUP_CODE_HASH_LENGTH=10
+##### Secret hash to encrypt invitations stored in application data base
+##### (Example: INVITATION_SALT=af21c49e35f01c1ec4d465daf75098e07a04c7ed)
 INVITATION_SALT=salt
-#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
-#TRUSTED_HOSTS=localhost,example.com
-###< symfony/framework-bundle ###
-
-###> symfony/swiftmailer-bundle ###
-# For Gmail as a transport, use: "gmail://username:password@localhost"
-# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
-# Delivery is disabled by default via "null://localhost"
-MAILER_URL=null://localhost
-DELIVERY_ADDRESS=your_dev@email.com
-APP_NAME='Caesar App'
-MAILER_PORT_1080=1080
-MAILER_PORT_1025=1025
-###< symfony/swiftmailer-bundle ###
-
-###> lexik/jwt-authentication-bundle ###
+##### Is a text used to control access to a jwt token
+##### Any word or random characters to add more entropy
 JWT_PASSPHRASE=
-###< lexik/jwt-authentication-bundle ###
 
-###> hwi/oauth-bundle ###
+#### Google Authentication (OAuth2)
+##### Creation of customer Google ID and secret code to authenticate google accounts
+##### described in official google documentation bellow by link
+##### https://developers.google.com/adwords/api/docs/guides/authentication?hl=ru#create_a_client_id_and_client_secret
 GOOGLE_ID=id
 GOOGLE_SECRET=secret
-###< hwi/oauth-bundle ###
+##### Allowed urls to redirect by Google authentication
+ALLOW_FRONT_REDIRECT_PATTERN='/^http[s]*:\/\/(example.com|example2.com)/'
+
+#### Email System
+##### For Gmail as a transport, use: "gmail://username:password@localhost"
+##### For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+##### Delivery is disabled by default via "null://localhost"
+MAILER_URL=null://localhost
+
+#############################
+## Optional variables
+#### App environment
+
+##### Is Application title
+#APP_NAME='Caesar App'
+
+#### Encryption difficulty setting by length
+#BACKUP_CODE_HASH_LENGTH=10
+##### Additional permissions to enter the application
+#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
+#TRUSTED_HOSTS=localhost,example.com
+
+#### This value is used to set the sender's address in emails
+#SENDER_ADDRESS=noemail@example.com
+#DB_VERSION=9.6
 
-#Setup your project path to open files by link from browser https://symfony.com/doc/current/reference/configuration/framework.html#ide
-#For Phpstorm use such string: phpstorm://open?url=file://%f&line=%l&/var/www/html/>/path/to/project/on/host/machine
-IDE_STRING=''
+#### Redis version 5.0.4
+#REDIS_HOST=redis
+#REDIS_PORT=6379
 
-###> snc/redis-bundle ###
-REDIS_HOST=redis
-REDIS_PORT=6379
-###< snc/redis-bundle ###
-###> rabbitMQ ###
-RABBITMQ_HOST=rabbitmq
-RABBITMQ_VHOST=/
-RABBITMQ_DEFAULT_USER=rabbitmq
-RABBITMQ_DEFAULT_PASS=rabbitmq
-###< rabbitMQ ###
\ No newline at end of file
+#### RabbitMQ version 3.7.14
```

Comment:
Is RabbitMQ service optional? @mg-4xxi

Reply:
just already has default values
this is important for security but not important to deploy the project

Reply:
I do not agree. Without this service, sending mail will not work for you, and therefore, this service is just as important as a database.

Reply:
replaced